src/base/settingsstorage.cpp

```diff
@@ -56,8 +56,8 @@ namespace
         MappingTable(std::initializer_list<std::pair<QString, QString>> list)
         {
             reserve(static_cast<int>(list.size()));
-            for (std::initializer_list<std::pair<QString, QString>>::const_iterator it = list.begin(); it != list.end(); ++it)
-                insert(it->first, it->second);
+            for (const auto &i : list)
```

Comment:
Wait... 

>I would still not recommend to use range-based for with Qt containers.

agree.
But here the `for` loop is iterating over `std::initializer_list` which is not a Qt container.